widget/view.controller.js

```diff
@@ -815,10 +818,10 @@
           if (result && result['hydra:member'] && result['hydra:member'].length > 0) {
             $scope.socResult.automatedClosed = result['hydra:member'][0].status;
           }
-          if($scope.socResult.closedAlerts === 0){
+          if ($scope.socResult.closedAlerts === 0) {
             var closedAutomatedAlerts = '(0%';
```

Comment:
please always use const for hard coded strings like (0%

Reply:
Done